.github/workflows/push.yml

```diff
@@ -189,7 +189,7 @@ jobs:
     needs:
       - build_and_push
       - branch_meta
-    uses: hpi-schul-cloud/end-to-end-tests/.github/workflows/e2e_call.yml@main
+    uses: hpi-schul-cloud/end-to-end-tests/.github/workflows/e2e_call.yml@BC-7461-mongo6
```

Comment:
this was added only so that the e2e tests called from server repo pass
it must be removed before merge